docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -115,6 +115,12 @@ catalog configuration.
     - Value of the new client (application) secret created
   * - ``hive.azure.adl-refresh-url``
     - OAuth 2.0 token endpoint url
+  * - ``hive.azure.adl-proxy-host``
+    - Proxy host and port in ``host:port`` format. Use this property to connect
+      to an ADLS endpoint via an internet proxy from a host from within an
```

Comment:
ADLS or ADL?

Reply:
ADLS as per conversations

Reply:
@losipiuk - can you weigh in on this? Slack convo referred to ADLS.

Reply:
internet -> SOCKS